### DISCUSSION TITLE
Function to like downloaded posts

### DISCUSSION THREAD

**Discussion by ThickenWing** [DIGITALCRIMINALS/OnlyFans#651] [Ideas]
Apologies if this is the incorrect area in which to discuss this, but I'd like to ask if it'd be possible to consider adding a function to like posts either as a standalone function or alongside a download? The likes help out the models more than we'd care to think and it seems like it'd be low-hanging fruit for such adept programmer(s).

**Comment by OFfriend** [comment]
you can like only 1000 posts per day

**Reply by ThickenWing** [reply]
> you can like only 1000 posts per day
I didn't know that, but that is fair. All things considered, I wouldn't turn down the opportunity to formally like at least some of the content creators whom I support in an automated manner. I'd try myself but I recognize when there are far-more-gifted coders present. Python is like wizardry to my rookie, dotnet arse.

**Reply by ThickenWing** [reply]
I can dig it. 1k is a finite limit, I'm comfortable with that as I'm subbed to a small number of Content Creators, but I'd certainly like to flirt with the possibility of making my scraping and enjoying their content worth the time. Granted things aren't as tough as the once-were, but that is beside the point. I merely wish to like as many posts as I can, first 1k come, first 1k served, I suppose. It may not seem like much, but it matters more than I've seen in discussions. A better support and subsequent bottom-line leads, inexorably to better/more content.
